VirtoCommerce.Platform.Web/Controllers/HomeController.cs

```diff
@@ -41,12 +50,15 @@ public ActionResult Index()
                 }
             }
 
+            var favIcon = GetFavIcon();
```

Comment:
Seems it is redundant variable,  GetFavIcon() can be called in place.

Reply:
Done